CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.7.4] - 2020-07-29
+
 - Adjust MAX_PODS for master and worker nodes to max IP's per ENI when using aws-cni
```

Comment:
There should be some header. Not sure if changed or fixed. 

```suggestion
### Fixed 

- Adjust MAX_PODS for master and worker nodes to max IP's per ENI when using aws-cni
```

Reply:
Hmm I cannot change the commit. Shall I just create a new? Apply the "Fixed" header in the master and then l recreate the release? WDYT?

![image](https://user-images.githubusercontent.com/1936982/88788160-3ce20880-d195-11ea-8c31-fbbada685449.png)